channeldb/invoice_test.go

```diff
@@ -147,43 +147,6 @@ func TestInvoiceWorkflow(t *testing.T) {
 	if _, err := db.LookupInvoice(fakeHash); err != ErrInvoiceNotFound {
 		t.Fatalf("lookup should have failed, instead %v", err)
 	}
-
-	// Add 10 random invoices.
-	const numInvoices = 10
-	amt := lnwire.NewMSatFromSatoshis(1000)
-	invoices := make([]*Invoice, numInvoices+1)
-	invoices[0] = &dbInvoice2
-	for i := 1; i < len(invoices)-1; i++ {
-		invoice, err := randInvoice(amt)
-		if err != nil {
-			t.Fatalf("unable to create invoice: %v", err)
-		}
-
-		hash := invoice.Terms.PaymentPreimage.Hash()
-		if _, err := db.AddInvoice(invoice, hash); err != nil {
-			t.Fatalf("unable to add invoice %v", err)
-		}
-
-		invoices[i] = invoice
-	}
-
-	// Perform a scan to collect all the active invoices.
-	dbInvoices, err := db.FetchAllInvoices(false)
-	if err != nil {
-		t.Fatalf("unable to fetch all invoices: %v", err)
-	}
-
-	// The retrieve list of invoices should be identical as since we're
-	// using big endian, the invoices should be retrieved in ascending
-	// order (and the primary key should be incremented with each
-	// insertion).
-	for i := 0; i < len(invoices)-1; i++ {
-		if !reflect.DeepEqual(*invoices[i], dbInvoices[i]) {
-			t.Fatalf("retrieved invoices don't match %v vs %v",
-				spew.Sdump(invoices[i]),
-				spew.Sdump(dbInvoices[i]))
-		}
-	}
```

Comment:
No because it tested that the `FetchAllInvoices` retrieves invoices added with `AddInvoice`. `FetchAllInvoices` is dead code and removed, and `AddInvoices` has coverage in other tests.

Reply:
Good point